### ISSUE TITLE
Recent failing

### ISSUE THREAD

**Issue by waghanza** [mahmoud/clastic#32]
Hi @mahmoud,

Using `clastic` **20.0**, I have

```python
Traceback (most recent call last):
  File "/usr/src/app/server.py", line 6, in <module>
    from clastic import Application, render_basic
  File "/usr/local/lib/python3.9/site-packages/clastic/__init__.py", line 20, in <module>
    from . import server
  File "/usr/local/lib/python3.9/site-packages/clastic/server.py", line 18, in <module>
    from ._werkzeug_serving import reloader_loop, make_server
  File "/usr/local/lib/python3.9/site-packages/clastic/_werkzeug_serving.py", line 72, in <module>
    from werkzeug._compat import iteritems, PY2, reraise, text_type, \
ModuleNotFoundError: No module named 'werkzeug._compat'
```

Linked to : https://github.com/the-benchmarker/web-frameworks/issues/4281

Regards,

**Comment by mahmoud** [comment]
Ah, yeah, werkzeug just released 2.0, I'll need to repin and rerelease. Thanks for the heads up.

**Comment by mahmoud** [comment]
@waghanza Fixed now (in 21.0.0), thanks again!

**Comment by waghanza** [comment]
solved in https://the-benchmarker.semaphoreci.com/jobs/58bbfb38-d9db-40e4-bbe6-e60690dbb79a. thx
